corehq/apps/app_manager/views/releases.py

```diff
@@ -124,14 +125,29 @@ def _get_batch(start_build=None, skip=None):
             app_es = app_es.is_released()
         if build_comment:
             app_es = app_es.build_comment(build_comment)
+
         results = app_es.exclude_source().run()
+        total_apps = results.total
         app_ids = results.doc_ids
         apps = get_docs(Application.get_db(), app_ids)
+
+        if build_comment:
```

Comment:
Can also think about using `build_comment.is_digit()`?

Plus, i am always against extending already long methods, so if this can be extracted out, that would be great.

Reply:
`is_digit` returns true for floats, but I only want ints.

Not going to extract the method in this PR.

Reply:
I thought i checked it before suggesting. Tried again
```
In [1]: '1'.isdigit()
Out[1]: True

In [2]: '1.2'.isdigit()
Out[2]: False
```
Unless i am using a different version of python.

fine to ignore this comment.

Reply:
Huh, I thought I tried it and `0.2` came up `True`. Not going to go back and update this one, though.

Reply:
just confirmed that 0.2 also came out to be false. Just keeping note for future uses.